test/functional/apps/dashboard/group3/dashboard_state.ts

```diff
@@ -185,7 +185,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await PageObjects.dashboard.waitForRenderComplete();
     };
 
-    describe('Directly modifying url updates dashboard state', () => {
+    describe.skip('Directly modifying url updates dashboard state', () => {
```

Comment:
Can you add a comment on why test is skipped.

Reply:
Skipped as per @ThomThomson's direction, this is an underlying issue with React embeddables but not a high priority to fix

Reply:
Comment added but Github doesn't want to show it in the `/files` page for some reason. Committed here: https://github.com/elastic/kibana/pull/183197/commits/990185ed0975d081330f17299279afc36e428ac6

Reply:
The moment I posted that comment Github fixed itself

Normal website